src/vl-tooltip.js

```diff
@@ -253,6 +251,14 @@
         default:
       }
     }
+    
+    // supplement bin from fieldDef, user should never have to provide bin
+    if (fieldDef.bin) {
+      supplementedFieldOption.bin = true;
+      supplementedFieldOption.formatType = 'string'; // we show bin range as string (e.g. "5-10")
```

Comment:
It looks like in Vega `formatType` is `["time", "utc", "string", "number"]`

For axis: https://github.com/vega/vega/blob/master/src/parse/axes.js#L112
For legends: https://github.com/vega/vega/blob/master/src/parse/legends.js#L60

So I think I should leave `string` as a `formatType` option for tooltip also.